assets/site/guide.sass

```diff
@@ -142,15 +142,13 @@ html.cockpit-guide
 
   h2, h3, h4, h5, h6,
   dl.toc .part,
-  dl.toc .chapter,
     position: static
     padding: 1rem 0 0.5rem
 
-  h2,
-  dl.toc .part
-    font-size: 1.72rem
+  dl.toc .chapter
+    position: static
```

Comment:
This is to keep the same styling for the element as before but without the padding. So it is related to the commit message.